channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/dto/JaicpAnalyticsAPI.kt

```diff
@@ -0,0 +1,91 @@
+package com.justai.jaicf.channel.jaicp.dto
+
+import com.justai.jaicf.context.DefaultActionContext
+import com.justai.jaicf.context.ExecutionContext
+import kotlinx.serialization.Serializable
+
+/**
+ * Provides JAICP Analytics API for scenarios
+ *
+ * @see JaicpAnalyticsAPI
+ * */
+val DefaultActionContext.jaicpAnalytics: JaicpAnalyticsAPI
+    get() = reactions.executionContext.analyticsApi
+
+internal val ExecutionContext.analyticsApi: JaicpAnalyticsAPI
+    get() = (channelContext[ANALYTICS_BOT_CONTEXT_KEY] as? JaicpAnalyticsAPI ?: JaicpAnalyticsAPI()).also {
+        channelContext[ANALYTICS_BOT_CONTEXT_KEY] = it
+    }
+
+private const val ANALYTICS_BOT_CONTEXT_KEY = "com/justai/jaicf/jaicp/jaicpAnalyticsApi"
+
+@Serializable
+class JaicpAnalyticsAPI {
```

Comment:
Not obvious why the class with an "API" in its name acts like DTO. It would be better to separate concerns.

Reply:
Dunno. We already have `JaicpDialerAPI` which behaves like $dialer in JAICP. Now we have `JaicpAnalyticsAPI` which behaves like $analytics. Then we will have another API from another built-in jaicp service ($caila to `CailaAPI`, e.g.). 

I just dont feel like naming it JaicpAnalyticsDTO or just JaicpAnalytics.

@veptechno @nikvoloshin if you don't mind, I'd like to stick to this naming scheme.